bigquery/google/cloud/bigquery/job.py

```diff
@@ -1391,6 +1391,20 @@ def referenced_tables(self):
         return tables
 
     @property
+    def schema(self):
+        """Return schema from job statistics, if present.
+
+        See:
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#statistics.query.schema
+
+        :rtype: list of :class:`~google.cloud.bigquery.schema.SchemaField
```

Comment:
LGTM, but this reminds of an issue that has come up in some questions I've answered in GCP Slack of StackOverflow. Filed https://github.com/GoogleCloudPlatform/google-cloud-python/issues/3723 for a public way to get to/from the raw API representation of a schema.